core/sandbox/.circleci/config.yml

```diff
@@ -90,6 +90,16 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v\d+\.\d+\.\d+(-.+)?/
+      - tool-kit/publish:
```

Comment:
👀 why is this part added?

Reply:
It's a newer hook that's installed now, but this is just a test project so this config will never run and won't publish anything! 😁